### PR TITLE
Adds additional id parameters to subscribe_form and input email helpers

### PR DESCRIPTION
**Pull request by naz**
This PR addresses #9134

- adds form_id, input_id, and button_id parameters to subscribe_form helper
- adds id parameter to input_email helper
- adds test coverage for subscribe_form helper
- adds test coverage to input email helper

Would need to update https://themes.ghost.org/docs/subscribers#section-the-subscribe-form-helper when accepted

### CODE REVIEW

File: core/server/helpers/tpl/subscribe_form.hbs [hunk 1]
```diff
@@ -1,11 +1,11 @@
-<form method="post" action="{{action}}" class="{{form_class}}">
+<form method="post" action="{{action}}" id={{form_id}} class="{{form_class}}">
```
Comment (kirrg001) [inline core/server/helpers/tpl/subscribe_form.hbs]: PR looks good to me :) Works great 👍 

I just wondered

> id={{form_id}} class="{{form_class}}"

We use double slashes and no double slashes. Looks like it doesn't matter? But maybe we should use double slashes for consistency reasons?

**Comment by kirrg001** [comment]
@gargol I remember you said, you want to add more tests? Let me know, then i'll wait with final review + merge.

**Comment by gargol** [comment]
@kirrg001 finished the main part of the issue. Helper tests need to be looked into, as it seems like template registry is not being cleared properly

**Comment by kirrg001** [comment]
Cool. Looks like first 5 commits belong together, don't they? And last one is separate, because it's unrelated to this bug fix. So we can merge 2 commits into master.

> Helper tests need to be looked into, as it seems like template registry is not being cleared properly

Will look at it if there is time later :)

**Review by kirrg001 — APPROVED** [review]

**Pull request merged**